src/Apps/Artist/ArtistApp.tsx

```diff
@@ -18,6 +19,8 @@ const ArtistApp: React.FC<React.PropsWithChildren<ArtistAppProps>> = ({
 }) => {
   useScrollToOpenArtistAuthModal({ name: artist.name })
 
+  const isCombinedPage = useIsRouteActive(`/artist/${artist.slug}/combined`)
```

Comment:
So this is the bit that will be replaced by an Unleash flag, yes?

Reply:
Yeah, exactly